goal_src/jak2/levels/fortress/ammo_dump/fort-robotank-turret.gc

```diff
@@ -1068,8 +1068,7 @@
        (set! (-> v1-22 prim-core collide-with) (collide-spec jak player-list))
        )
      (logior! (-> self flags) 128)
-     (format 0 "TODO - disabling roboscreen!~%")
-     (send-event (handle->process (-> self screen)) 'untrigger)
+     (send-event (handle->process (-> self screen)) 'trigger)
```

Comment:
Another one here (IIRC, we still need this because of texture anims)

Reply:
It seems to work 'fine' now.

![image](https://user-images.githubusercontent.com/7569514/234610088-2a3c76ce-5d8c-4037-aee7-cfe182c1312d.png)